Packs/Hoxhunt/Integrations/Hoxhunt/README.md

```diff
@@ -0,0 +1,204 @@
+# Hoxhunt
+
+Use the Hoxhunt integration to ingest user-reported phishing, email compromise, or other similar incidents.
+
+This integration is tested with **VERSION** of Hoxhunt's API. 
+
+## Configure Hoxhunt on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Hoxhunt.
+3. Click **Add instance** to create and configure a new integration instance.
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| api_url | Hoxhunt's API URL \(e.g., https://domain-to-api.com/) | True |
```

Comment:
Wonder if we could have default value for the api url and provided if needed for some testing etc.?